src/component/plotly-dash-preview/parse.js

```diff
@@ -0,0 +1,30 @@
+const isNonEmptyString = require('../../util/is-non-empty-string')
+
+/**
+ * @param {object} body : JSON-parsed request body
+ *  - url
+ *  - pdfOptions
+ * @param {object} opts : component options
+ * @param {function} sendToRenderer
+ * - errorCode
+ * - result
+ */
+function parse (body, opts, sendToRenderer) {
+  const result = {}
+
+  const errorOut = (code, msg) => {
+    result.msg = msg
+    sendToRenderer(code, result)
+  }
+
+  if (isNonEmptyString(body.url)) {
```

Comment:
We could probably be more strict here. Using `isUrl` (like done [here](https://github.com/plotly/image-exporter/blob/a7e67453807aa08c3b27dbdfb0e81934eab5c134/src/component/plotly-graph/inject.js#L57)) would be best.